tests/test_qubit_device_adjoint_jacobian.py

```diff
@@ -376,9 +408,8 @@ def circuit(x, y, z):
     thetas = np.linspace(-2 * np.pi, 2 * np.pi, 8)
 
     @pytest.mark.autograd
-    @pytest.mark.parametrize("reused_p", thetas**3 / 19)
-    @pytest.mark.parametrize("other_p", thetas**2 / 1)
-    def test_fanout_multiple_params(self, reused_p, other_p, tol, mocker, dev):
+    @pytest.mark.parametrize("all_p", list(zip(thetas**3 / 19, thetas**2 / 1)))
```

Comment:
Does this recover every combination? I feel like you have now grouped them pair wise and are iterating over each pair. Or was there some other motivation for this?

Reply:
Oh no! Very good catch, I was moving too quickly when linting this :grimacing:

Reply:
Changed to `product`.